modules/floating-buttons/documents/floating-buttons.php

```diff
@@ -97,8 +99,61 @@ public function filter_admin_row_actions( $actions ) {
 		return $built_with_elementor + $actions;
 	}
 
+	/**
+	 * Tries to find the post id of the floating element that is set as entire site.
+	 * If found, returns the post id, otherwise returns 0.
+	 *
+	 * @param string $floating_element_type
+	 *
+	 * @return int
+	 */
+	public static function get_set_as_entire_site_post_id( string $floating_element_type ): int {
+		static $types = [];
+
+		if ( isset( $types[ $floating_element_type ] ) ) {
+			return $types[ $floating_element_type ];
+		}
+
+		$query = new \WP_Query( [
+			'post_type' => Floating_Buttons_Module::CPT_FLOATING_BUTTONS,
+			'posts_per_page' => -1,
+			'post_status' => 'publish',
+			'fields' => 'ids',
+			'no_found_rows' => true,
+			'update_post_term_cache' => false,
+			'meta_query' => [
+				[
+					'key' => Module::FLOATING_ELEMENTS_TYPE_META_KEY,
+					'value' => $floating_element_type,
+				],
+				[
+					'key' => '_elementor_conditions',
+					'compare' => 'EXISTS',
+				],
+			],
+		] );
+
+		foreach ( $query->posts as $post_id ) {
```

Comment:
Add an early return in case `$query->posts` is not an array as expected here.

Reply:
@louiswol94 refactored to use get_posts() which should always return an array, if empty nothing happens and zero is returned at the end